src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractExecutableMemberWriter.java

```diff
@@ -205,95 +205,95 @@ public Boolean visitDeclared(DeclaredType t, Void unused) {
      * Add all the parameters for the executable member.
      *
      * @param member the member to write parameters for.
-     * @param htmltree the content tree to which the parameters information will be added.
+     * @param target the content to which the parameters information will be added.
      */
-    protected void addParameters(ExecutableElement member, Content htmltree) {
-        Content paramTree = getParameters(member, false);
-        if (paramTree.charCount() > 2) {
+    protected void addParameters(ExecutableElement member, Content target) {
+        Content params = getParameters(member, false);
+        if (params.charCount() > 2) {
             // only add <wbr> for non-empty parameters
-            htmltree.add(new HtmlTree(TagName.WBR));
+            target.add(new HtmlTree(TagName.WBR));
         }
-        htmltree.add(paramTree);
+        target.add(params);
     }
 
     /**
      * Add all the parameters for the executable member.
      *
      * @param member the member to write parameters for.
      * @param includeAnnotations true if annotation information needs to be added.
-     * @return the content tree containing the parameter information
+     * @return the content containing the parameter information
      */
     protected Content getParameters(ExecutableElement member, boolean includeAnnotations) {
-        Content paramTree = new ContentBuilder();
-        paramTree.add("(");
+        Content paramContent = new ContentBuilder();
```

Comment:
Continuing the theme of using more specific words than `content`, I've started using the word `result` for those cases where the code is building up an object that will be the result of the method.